services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitCloseForm.js

```diff
@@ -18,42 +18,33 @@ const propTypes = {
   title: PropTypes.string.isRequired,
 };
 
-const options = [
-  {
-    value: false,
-    label: "Open",
-  },
-  {
-    value: true,
-    label: "Closed",
-  },
-];
-
 export const ExplosivesPermitCloseForm = (props) => {
   return (
     <Form layout="vertical" onSubmit={props.handleSubmit}>
       <Row gutter={16}>
         <Col span={24}>
-          <Form.Item>
-            <Field
-              id="is_closed"
-              name="is_closed"
-              label="Permit Status*"
-              component={RenderRadioButtons}
-              customOptions={options}
-              validate={[requiredRadioButton]}
-            />
-          </Form.Item>
+          <Typography.Title level={2}>Close Permit</Typography.Title>
+          <Typography.Paragraph>
+            If you perform this action there will no longer be an active explosive use and storage
+            permit for this notice of work. If you still want to have an amendment active, consider
+            creating an amendment for this permit before performing this action.
+          </Typography.Paragraph>
         </Col>
       </Row>
       <Row gutter={16}>
         <Col span={24}>
-          <Form.Item>
+          <Form.Item label="Date Permit was Closed" required>
             <Field
               id="closed_timestamp"
               name="closed_timestamp"
-              label="Date Permit was Closed"
-              component={RenderDate}
+              normalize={normalizeDatetime}
+              component={RenderDateTimeTz}
+              validate={[
+                dateNotInFutureTZ,
+                required,
+                dateTimezoneRequired("esup_permit_close_timezone"),
+              ]}
+              props={{ timezoneFieldProps: { name: "esup_permit_close_timezone" } }}
```

Comment:
In hindsight, all this complexity could have been avoided if we saved if we saved it as a "YYYY-MM-DD" type string instead. But I do like the normalize function, isn't it cool?

Reply:
😁 So cool